src/poprox_recommender/topics.py

```diff
@@ -43,6 +76,19 @@ def user_topic_preference(past_articles: list[Article], click_history: list[Clic
     return topic_count_dict
 
 
+def user_locality_preference(past_articles: list[Article], click_history: list[Click]) -> dict[str, int]:
```

Comment:
Do we want to calibrate on the user's locality preference? I thought we might want to calibrate on the distribution of today's articles or a fixed distribution from our prior investigation.

Notably, I see an issue at the cold start of our experiment where users might have a very focused click history due to the focus of the baseline POPROX algorithm and their locality preference will be very biases.

Reply:
Could you elaborate on what we might want to calibrate on in this function? I was thinking it as just a function to compute the "groud_truth" of user click history with the system, and the actual calibration step happened after we get their preference (in the new `LocalityCalibration` class). 

The cold-start issue might exist, but if we have an one-month window to observe users and collect their preferences, can that be a feasible solution?